docs/DeveloperGuide.adoc

```diff
@@ -800,18 +801,63 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new tutor/student |
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |user |delete a tutor/student |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a tutor/student by name |locate details of persons without having to go through the entire list
+
+|`* * *` |user |edit the <<criteria, criteria>> of a tutor/student |change criteria without re-adding the entry
+
+|`* * *` |user |list all tutors or students separately |view a cleaner list without mixing tutors and students
+
+|`* * *` |user |set a tutor/student’s <<status, status>> |know who is available
+
+|`* * *` |user |mark a tutor/student as <<matched, matched>> with some other tutor/student for a certain subject |identify tutor/student status
 
 |`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
 
+|`* *` |user |protect the app with a password |ensure confidentiality of the contacts
+
+|`* *` |user |tag a person with role, subject, level, and status |identify details of each tutor/student easily
+
+|`* *` |user |sort matching list by budget |find the best match at the lowest price
+
+|`* *` |user |rate a tutor according to student’s feedback |identify tutors’ capability for future references
+
+|`* *` |user |store a list of all tutor/student that have been matched |I can have a cleaner list consisting of only unmatched tutor/student and also able to refer to past records if required
```

Comment:
you may want to reword this one slightly. this gives an impression that you won't store the unmatched tutor/students